content/articles/how-sweet-is-kubernetes/index.md

```diff
@@ -0,0 +1,289 @@
+---
+layout: engineering-education
+status: publish
+published:true 
+url: /engineering-education/how-sweet-is-kubernetes/
+title: How sweet is Kubernetes?
+description: How sweet is Kubernetes is an article which is targeted at simplifying concepts of Kubernetes for engineering students and aspiring DevOps professionals. It started with demystifying the basic concept of micro-service using Fleetman Tracking Application which is built on a simple micro-service architecture, using it as a tool to further explain the concepts of Kubernetes in the simplest manner. The tutorial will serve as a good foundation for beginners and also a means of enlightenment for intermediate-level developers and junior DevOps engineers.
+Grab a cup of coffee and enjoy the wonderful piece of work.
+author: qoyum-yusuf
+date: 2021-05-01T00:00:00-08:00
+topics: [HOW SWEET IS KUBERNETES?]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/quoyum-yusuf/hero.jpeg
+    alt: How sweet is Kubernetes cover image 
+---
+
+
+
+
+### HOW SWEET IS KUBERNETES?
+
+![kubernetes.jpeg](hero.jpeg)
```

Comment:
Hey I wanted you to rename the file itself like how you renamed your selfie.jpg to avatar.jpg. I wanted you to do the same with this. Rename kubernetes.jpeg to hero.jpg.